Assets/HoloToolkit/BuildAndDeploy/Editor/BuildSLNUtilities.cs

```diff
@@ -126,32 +136,46 @@ public static void PerformBuild(BuildInfo buildInfo)
                 PlayerSettings.SetScriptingDefineSymbolsForGroup(buildTargetGroup, buildInfo.BuildSymbols);
             }
 
-            string buildError = "Error";
-            try
+            // For the WSA player, Unity builds into a target directory.
+            // For other players, the OutputPath parameter indicates the
+            // path to the target executable to build.
+            if (buildInfo.BuildTarget == BuildTarget.WSAPlayer)
             {
-                // For the WSA player, Unity builds into a target directory.
-                // For other players, the OutputPath parameter indicates the
-                // path to the target executable to build.
-                if (buildInfo.BuildTarget == BuildTarget.WSAPlayer)
-                {
-                    Directory.CreateDirectory(buildInfo.OutputDirectory);
-                }
+                Directory.CreateDirectory(buildInfo.OutputDirectory);
+            }
+
+            OnPreProcessBuild(buildInfo);
 
-                OnPreProcessBuild(buildInfo);
-                buildError = BuildPipeline.BuildPlayer(
+            EditorUtility.DisplayProgressBar("Build Pipeline", "Herding all the cats...", 0.25f);
```

Comment:
Do we really want "Herding all the cats"? One of our release criteria is globalization and I worry that this may be misunderstood / found offensive.

Reply:
Ahh, sorry about that. It wasn't meant to stay nor be offensive in any way. It was just a way of saying we're gathering all the data.

I'll be sure to update this.

Reply:
No stress. Just me being paranoid :)